packages/card/src/Card.ts

```diff
@@ -34,8 +34,6 @@ import '@spectrum-web-components/checkbox/sp-checkbox.js';
 import '@spectrum-web-components/popover/sp-popover.js';
 import '@spectrum-web-components/divider/sp-divider.js';
 import cardStyles from './card.css.js';
-import headingStyles from '@spectrum-web-components/styles/heading.js';
```

Comment:
This is duplicative - the card styles provide the typography needed for the heading and subtitle sections.